klondikemarlen/legal-files-integration
# Issue 1: Create an Microsoft SQL Server integration

Fixes [#1](https://github.com/klondikemarlen/legal-files-integration/issues/1)

# User Story
A Drupal form A can be posted to service and the data is stored on a per field basis in a Microsoft SQL Server database table.

# Implementation Details

- Use [Sequelize](https://sequelize.org/) as to generate migrations, models and connect to the database.

# Testing Instructions

1. Boot the app via `dev up --build`.
2. Post example data to the `/api/form-a` endpoint.

```bash
curl --request POST localhost:3000/api/form-a \
  --header "Content-Type: application/json" \
  --data '
{
  "firstName": "test",
  "email": "linuxize@example.com"
}
'
```

3. Check that the returned data is correct. It should look like:

```
{"success":true}
```
